### PR TITLE
Added colors output to tests results and round is almost over

### LINKED ISSUE #31: Add color to text output
To improve the usability all the text output should be colored. This was based on @douglasshimada #28 PR, thx!

Tests passed message could be green
Failed tests message red (only the header is red, the traceback and other details are "black")

This could be done using [colorama](https://github.com/tartley/colorama) lib that color texts from all operating systems.

Any other ideas?

### PR DESCRIPTION
**Pull request by heitorPB**
Yellow color: 'Round is going to finish in 60 seconds'
Red color: '\nTests failed!\n'
Green color: '\nTests passed!\n'

Fixes #31

### CODE REVIEW

**Review by zsinx6 — CHANGES_REQUESTED** [review]
@heitorPB the branch is outdated, can you fix to be merged?

File: dojo_toolkit/dojo.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from dojo_toolkit.test_runner import DoctestTestRunner
 from dojo_toolkit.timer import Timer
 
+from clint.textui import colored
```
Comment (zsinx6) [inline dojo_toolkit/dojo.py:13]:
```suggestion
from clint.textui import colored

```

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,6 @@
 pgi==0.0.11.1
 watchdog==0.10.3
 pyglet==1.2.4
+six==1.10.0
+mock==2.0.0
+clint
```
Comment (zsinx6) [inline requirements.txt:6]: Why not use a fixed version for the clint?

File: requirements.txt [hunk 3]
```diff
@@ -1,3 +1,6 @@
 pgi==0.0.11.1
 watchdog==0.10.3
 pyglet==1.2.4
+six==1.10.0
+mock==2.0.0
```
Comment (zsinx6) [inline requirements.txt:5]: Is `six` and `mock` really needed?

Comment (heitorPB) [reply requirements.txt:5]: I made this code 2.5 years ago. I have no idea.

Comment (zsinx6) [reply requirements.txt:5]: Both can be removed

File: requirements.txt (hunk 2 above)
Comment (zsinx6) [reply requirements.txt:6]:
```suggestion
clint==0.5.1
```

File: requirements.txt (hunk 3 above)
Comment (zsinx6) [inline requirements.txt:5]:
```suggestion
```

Comment (zsinx6) [reply requirements.txt:5]:
```suggestion
```

**Review by zsinx6 — DISMISSED** [review]

**Comment by zsinx6** [comment]
Just rebased this, and will merge

**Review by zsinx6 — APPROVED** [review]

**Pull request merged**